sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/function/Endpoint.java

```diff
@@ -0,0 +1,53 @@
+package com.alibaba.csp.sentinel.util.function;
```

Comment:
😅Is it better to put them in transport-common instead of the core package?
Or are there other pieces of code outside the transport module depending on them?

Reply:
And please `function` makes little sense here. I believe you really can figure out a good naming💪

Reply:
It's my mistake, I'll move them to `com.alibaba.csp.sentinel.transport.endpoint`.